engine/src/main/java/org/hibernate/validator/constraints/ScriptAssert.java

```diff
@@ -111,6 +111,14 @@
 	String alias() default "_this";
 
 	/**
+	 * @since 5.4
+	 * @return The name of the property on which you would like to report a validation error for.
```

Comment:
- s/on which/for which/
- remove the for at the end of the sentence
- the `@since 5.4` should be after the `@return`